git-repo-manager/src/service.rs

```diff
@@ -39,8 +47,16 @@ impl GitRepoManager for GitRepoManagerService {
             self.config.git_repository_base_path, request_data.repository_path
         );
 
+        // Log
+        info!("Creating new repository : {}", new_repo_full_path);
+
         // Check if the repository already exists, if it does return error otherwise continue
-        if fs::metadata(new_repo_full_path.clone()).is_ok() {
+        if fs::metadata(&new_repo_full_path).is_ok() {
+            // Log
+            error!(
```

Comment:
Errors are logged when something requires immediate attention. However, a repo already existing is expected behavior. I'd say it would be an info log at best.

Reply:
all of them are now deleted :+1: